### ISSUE TITLE
Won't launch on XP SP3

### ISSUE THREAD

**Issue by DurhamDecLab** [rafalh/dashfaction#115]
Hello, possibly I'm the only one trying to get this to work on XP SP3? Installs fine, won't launch with error as per screenshot. P4, 3GB RAM.

![df](https://user-images.githubusercontent.com/46998441/122649032-d3a81280-d123-11eb-8423-11e1d35b3141.JPG)

**Comment by mjr4077au** [comment]
Have you tried compiling it with MSVC 2017? It'd probably build OK, I can only assume the project is compiled against MSVC 2019 which dropped XP.

**Comment by DurhamDecLab** [comment]
I haven't tried that, I'll have a look into it. Thanks for the reply.

**Comment by rafalh** [comment]
Official Dash Faction versions are compiled using MinGW, not MSVC so it's not that. This incompatibility is actual usage of API that doesn't exist in Win XP: https://github.com/rafalh/dashfaction/blob/92c66e2749b35ead2ede6401e1e1bda8ab797e57/game_patch/os/win32_console.cpp#L100
I do not want to invest time into supporting a 20 year old operating system. I changed README so now DF officially supports Win 7+. 
If you want to use it on Win XP anyway use Dash Faction 1.6.1 or remove/comment out usage of the `GetFinalPathNameByHandleA` API that I linked above and rebuild using a compiler that supports Win XP.
